air-script/tests/random_values/random_values.air

```diff
@@ -0,0 +1,18 @@
+def RandomValuesAir
+
+trace_columns:
+    main: [a, b]
+    aux: [c, d]
+
+public_inputs:
+    stack_inputs: [16]
+
+random_values:
+    rand: [x, y[14], z]
+
+integrity_constraints:
+    enf c' = z - $rand[0] + y[2]
+
+boundary_constraints:
+    enf c.first = $rand[5] + y[2] + z
+    enf c.last = x + $rand[15] + y[10]
```

Comment:
This test looks great.

nit: let's put the `boundary_constraints` before the `integrity_constraints`, since we usually use that order
nit 2: maybe we should use a different name instead of `rand`, like `alphas`, since these tests also help serve as examples/documentation of functionality.

Reply:
I noticed in the inline `TODO` below that `alphas` won't be supported yet in the IR, so feel free to ignore "nit 2" for now. Maybe we can update this when the other change is made